templates/search/tag_with_label.html

```diff
@@ -0,0 +1,20 @@
+<span class="{{ extra_classes }}">
+    <input type="checkbox"
+           id="{{ slug }}"
+           name="{{ input_name }}"
+           class="hidden"
+           value="{{ slug }}"
+           {% if slug in params %}checked{% endif %}
+           autocomplete="off">
+    <label class="fr-label invisible ml-0" for="{{ slug }}"></label>
+    <button class="fr-tag a4a-label-tag mb-1"
+        aria-pressed="{% if slug in params %}true{% else %}false{% endif %}"
+        data-fr-js-toggle="true"
+        type="button"
+        data-event-type="{% if group %}groupLabelClicked{% else %}filterAdded{% endif %}"
```

Comment:
This could / should be handled in another way to keep the HTML generic, maybe just pass the string of the expected event type